test/support/mock-crypto-data.js

```diff
@@ -26,6 +26,19 @@ module.exports = function createCryptoDataMock() {
     next();
   });
 
+  v1beta1.get(
+    "/crypto/:symbol/snapshot",
+    apiMethod((req) => {
+      if (req.params.symbol == null) {
+        throw apiError(422);
+      }
+      if (req.query.exchange == null) {
+        throw apiError(422);
```

Comment:
You can put these two condition into one if statement

Reply:
thanks for catching these :sweat_smile: Merged into one.